### PR TITLE
TWIS-34/Enter-code-screen

### PR DESCRIPTION
**Pull request by Akrrrrams**
TWIS-34/Enter-code-screen

## Link to Story.  

https://re-boot.atlassian.net/browse/TWIS-34

## Description

Please include a summary of the change and which issue is fixed.

## Screenshot(s) / GIF(s):
Screenshot 2024-03-06 at 1 28 43 PM

## Type of change

- [ ] Bug fix
- [x ] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [ x] My code follows the project's style guidelines
- [ x] I have created a Storybook story for my component(s)
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  //Function to be called on Resend email click
+  onResendEmail: () => void;
+}
+
+const EnterCodeScreen = ({
+  onResendEmail,
+}: HaventGotEmailYetProps): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="row" gap="1">
+      <Typography id="havent-got-the-email-yet" color={alpha("#000000", 0.46)}>
```
Comment (Jack-Vu) [inline src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx:14]: there is a space between the "Haven't got the email yet?" and "Resend email"

**Review by Jack-Vu — CHANGES_REQUESTED** [review]
The code looks good. But the spacing between the typography is a bit off. I also believe that the story criteria included the top portion as well just not the 6 boxes so this part
image

File: src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx (hunk 1 above)
Comment (Akrrrrams) [reply src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx:14]: that's right! I couldn't figure it out,  any idea how should I add that space?

File: src/app/_components/EnterCodeScreen/EnterCodeScreen.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  component: EnterCodeScreen,
+  parameters: {
+    layout: "centered",
+  },
+  tags: ["autodocs"],
+  argTypes: {},
+  decorators: [
+    (Story) => (
+      <Box width={"350px"}>
+        {}
```
Comment (Muradil-Erkin) [inline src/app/_components/EnterCodeScreen/EnterCodeScreen.stories.tsx:16]: magic brackets 👀

File: src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx (hunk 1 above)
Comment (Muradil-Erkin) [reply src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx:14]: padding or margin bottom

**Comment by Jack-Vu** [comment]
@Akrrrrams let us know when you finish up with component

**Comment by Muradil-Erkin** [comment]
@Akrrrrams we won't need this component anymore, closing this out.

**Pull request closed without merging**